README.md

```diff
@@ -20,9 +20,9 @@ Wagtail-Flags adds a Wagtail admin UI and Wagtail Site-based condition on top of
 
 ## Dependencies
 
-- Python 2.7+, 3.6+
-- Django 1.11+, 2.0+
-- Wagtail 1.13+, 2.0+
+- Python 3.6, 3.8
```

Comment:
I believe that we decided (in some meeting in recent months) that we don't want our READMEs and `setup.py`s to specify which minor version of Python 3 we're supporting. We don't want to have to update our docs every time a new minor version is released. So I think this line should be: 

```suggestion
- Python 3
```